init.lua

```diff
@@ -22,111 +41,53 @@ minetest.register_chatcommand("nametag",{
-		end
+	elseif colors[param] then
+		player:set_nametag_attributes({
+			color = colors[param]
+})
+minetest.chat_send_player(user, "Your nametag color has been set to: " .. param)
+minetest.log("action", (user.. " changed their nametag color to: " .. param))
+	else
+		minetest.chat_send_player(user, "Invalid usage, see /help nametag")
+		minetest.log("action", (user .. " used invalid arguments to change their nametag color"))
 	end
+end
 })
 
-minetest.register_chatcommand("nametag-custom", {
-	params = "<Red Value> <Green Value> <Blue Value>",
-	description = "Change the colour of a nametage to a custom colour (in RGB Format)",
+-- Customized RGB-format color
+minetest.register_chatcommand("custom-nametag", {
+	params = "<red> <green> <blue>",
+	description = "Customizable-nametag color",
+	privs = {nametag = true},
 	func = function(user, param)
 		local found, _, redValue, greenValue, blueValue = param:find("^([^%s]+)%s+(.+)%s+(.+)$")
 		if not found then
-			minetest.chat_send_player(user, "Not enough Arguments!")
+			minetest.chat_send_player(user, "Invalid usage, see /help custom-nametag")
+			minetest.log("action", (user .. " used invalid arguments to change their nametag color"))
 			return
 		end
 		local player = minetest.get_player_by_name(user)
 		player:set_nametag_attributes({
 			color = {r = redValue, g = greenValue, b = blueValue}
 		})
-		minetest.chat_send_player(user, "Your nametag colour has been set to the RBG value of " ..redValue .. ", " .. greenValue .. ", " .. blueValue)
+		minetest.chat_send_player(user, "Your nametag color has been set to the value of: " ..redValue .. ", " .. greenValue .. ", " .. blueValue)
+		minetest.log("action", (user .. " changed their nametag color using RGB-format to: "..redValue .. ", " .. greenValue .. ", " .. blueValue))
 	end
 })
 
 minetest.register_on_player_receive_fields(function(player, formname, fields)
 	if formname ~= "name-colours" then return end
-	if fields.green then
-		player:set_nametag_attributes({
-			color = {r = 0, g = 255, b = 0}
-		})
-	elseif fields.blue then
-		player:set_nametag_attributes({
-			color = {r = 0, g = 0, b = 255}
-		})
-	elseif fields.red then
-		player:set_nametag_attributes({
-			color = {r = 255, g = 0, b = 0}
-		})
-	elseif fields.purple then
-		player:set_nametag_attributes({
-			color = {r = 200, g = 0, b = 200}
-		})
-	elseif fields.yellow then
-		player:set_nametag_attributes({
-			color = {r = 255, g = 255, b = 0}
-		})
-	elseif fields.cyan then
-		player:set_nametag_attributes({
-			color = {r = 0, g = 255, b = 255}
-		})
-	elseif fields.white then
-		player:set_nametag_attributes({
-			color = {r = 255, g = 255, b = 255}
-		})
-	elseif fields.orange then
-		player:set_nametag_attributes({
-			color = {r = 255, g = 165, b = 0}
-		})
-	elseif fields.black then
-		player:set_nametag_attributes({
-			color = {r = 0, g = 0, b = 0}
-		})
-	end
+	for key, value in pairs(colors) do
+   	 if fields[key] then
```

Comment:
Superfluous leading space and wrong indents below.

Reply:
How may I improve it? 🙂

Reply:
Just remove 'Superfluous leading space' and fix 'wrong indents below' 🙂